lib/registration.pm

```diff
@@ -150,6 +150,10 @@ sub add_suseconnect_product {
     $version //= '${VERSION_ID}';
     $arch    //= '${CPU}';
     $params  //= '';
+    $timeout //= 300;
+    return 0 unless script_run("SUSEConnect -p $name/$version/$arch $params", $timeout);
+    return unless ($name =~ /PackageHub/ && is_sle '>=15');
```

Comment:
This is still too obfuscated code and anti-best-practices to fulfill my code quality expectations.
Sorry, I cannot merge this.